### ISSUE TITLE
Lost the iSpindel reading and data 

### ISSUE THREAD

**Issue by terragady** [vitotai/BrewPiLess#315]
![obraz](https://user-images.githubusercontent.com/19210041/108294106-80ce4680-7195-11eb-887a-3fa9f521d937.png)
![obraz](https://user-images.githubusercontent.com/19210041/108294125-8af04500-7195-11eb-899e-c3059e654080.png)

I have lost an iSpindel Column on the main screen, it is updating data because I can see the AUX temp updating with correct value. What is wrong? I flashed firmware again and formatted spiffs etc.

**Comment by vitotai** [comment]
Yes. in 4.1. The iSpindle Enable must be ON to show iSpindel on the main page.
